### PR TITLE
add homebrew cask support

### PR DESCRIPTION
**Pull request by KarlZeo**
## PR Checklist
* [ ] Closes #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Requires FONTLOG.txt to be updated
* [ ] Requires [/images/cascadia-code.png](/microsoft/cascadia-code/blob/master/images/cascadia-code.png) and/or [/images/cascadia-code-characters.png](/microsoft/cascadia-code/blob/master/images/cascadia-code-characters.png) to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Validation Steps Performed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ Clicking on the Cascadia.ttf file will download it to your machine. From there,
 
 👉 **Note:** If you have previously installed a version of Cascadia Code, installing a new version *should* overwrite the previous version. However, in order to ensure it is completely overwritten, it is recommended that you delete the previous version you had before installing another.
 
+**If you use macOS,you can install it by [Homebrew](https://brew.sh/)**
+
+```
+brew tap homebrew/cask-fonts 
+brew brew cask install font-cascadia
```
Comment (sepehr) [inline README.md]: Sounds like an extra `brew` there.

Comment (KarlZeo) [reply README.md]: emmmmmmm
Ok, I fix it.

**Comment by KarlZeo** [comment]
@sepehr 
Please check it again

**Comment by thedanfernandez** [comment]
Hey folks, this is a small change to a readme.md file, what's the delay?

**Comment by KarlZeo** [comment]
@thedanfernandez Now,on macOS,we can install this font by tow commands.

like this.

```
brew tap homebrew/cask-fonts 
brew cask install font-cascadia
```

I think every macOS user want this.

**Comment by mobeigi** [comment]
Quickstart Mac instructions with brew are great. Should definitely be exposed in the readme in my opinion.

**Comment by mittalyashu** [comment]
Any updates on support for homebrew?

**Pull request closed without merging**